config/config.md

```diff
@@ -5,6 +5,22 @@
 checker_severity_map.json file contains a mapping between a
 checker name and a severity level. Severity levels can be found in the shared.thrift file.
 
+The following severity levels are defined:
```

Comment:
Please format the code by using list and use bold text for severity level names:
```
- **STYLE**: A true positive indicates that the source code is against a specific coding guideline or could improve readability.
*Example*: LLVM Coding Guideline: Do not use else or else if after something that interrupts control flow (break, return, throw, continue).

- **LOW**: A true positive indicates that the source code is hard to read/understand or could be easily optimized.
*Example*: Unused variables, Dead code.

- **MEDIUM**: A true positive indicates that the source code that may not cause a run-time error (yet), but against intuition and hence prone to error.
*Example*: Redundant expression in a condition.

- **HIGH**: A true positive indicates that the source code will cause a run-time error.
*Example*: out of bounds array access, division by zero, memory leak.

- **CRITICAL**: Currently unused. This severity level is reserved for later use.
```

Otherwise LGTM!